VultisigApp/VultisigApp/Chains/Solana.swift

```diff
@@ -23,6 +23,9 @@ enum SolanaHelper {
             throw HelperError.runtimeError("fail to get to address")
         }
         
+        let priorityFeePrice = 1_000_000; // Turbo fee in lamports, around 5 cents
```

Comment:
this hardcode the priority fee , previous we get it from the block chain , are you sure this is the right fix?

Reply:
I had to. The blockchain returns a very low one.